zul/src/org/zkoss/zul/Slider.java

```diff
@@ -458,6 +462,48 @@ public void setRange(double minpos, double maxpos) {
 		setMaxpos(maxpos);
 	}
 
+	/** Sets the angleArc.
```

Comment:
1. Add some description like: Meaningful only used with knob mold.
2. angleArc should be replaced with a better words, like a degree of arc

Reply:
ok